This file is part of mingw-cross-env.
See doc/index.html for further information.

From b3ade9a63e881e69b4c9cfe7b5dbad78dcc4a0e0 Mon Sep 17 00:00:00 2001
From: Peter Stuge <peter@stuge.se>
Date: Wed, 1 Feb 2012 09:53:44 +0100
Subject: [PATCH] Fix undefined reference to _libssh_error in libgcrypt
 backend

Commit 209de22299b4b58e582891dfba70f57e1e0492db introduced a function
call to a non-existing function, and since then the libgcrypt backend
has not been buildable.
---
 src/libgcrypt.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/libgcrypt.c b/src/libgcrypt.c
index 1bda5ee..5c2787b 100644
--- a/src/libgcrypt.c
+++ b/src/libgcrypt.c
@@ -581,7 +581,7 @@ _libssh2_pub_priv_keyfile(LIBSSH2_SESSION *session,
                           const char *privatekey,
                           const char *passphrase)
 {
-    return _libssh_error(session, LIBSSH2_ERROR_FILE,
+    return _libssh2_error(session, LIBSSH2_ERROR_FILE,
                          "Unable to extract public key from private key file: "
                          "Method unimplemented in libgcrypt backend");
 }
-- 
1.7.8.3

